mr/impl/table_impl.h

```diff
@@ -180,6 +180,9 @@ class TableBase : public std::enable_shared_from_this<TableBase> {
     is_identity_ = false;
   }
 
+  // NOTE(ORI): CreateHandler binds a handler to fiber-local data, make sure
```

Comment:
no need to write `note(ORI):` imho. just write the comment. Also, please follow doxygen convention.
you can find how it [looks here for example](https://romange.github.io/gaia/classutil_1_1IoContextPool.html).

Reply:
Also holds for other comments you added in the PR